pkgs/development/python-modules/servefile/default.nix

```
@@ -0,0 +1,34 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pyopenssl
+, pytestCheckHook
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "servefile";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "sebageek";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-/ZEMZIH/ImuZ2gh5bwB0FlaWnG/ELxfBGEJ2SuNSEb8=";
+  };
+
+  propagatedBuildInputs = [ pyopenssl ];
+
+  checkInputs = [ pytestCheckHook requests ];
+  # Test attempts to connect to a port on localhost which fails in nix build
+  # environment.
+  disabledTests = [ "test_abort_download" ];
```

```suggestion
  disabledTests = [ "test_abort_download" ];
```

need to add two more:
```
       > FAILED tests/test_servefile.py::test_https - requests.exceptions.ConnectionEr...
       > FAILED tests/test_servefile.py::test_https_big_download - requests.exceptions...

---

what platform are you running on? is sandboxing enabled? I'm not seeing these errors but they look like similar to the ones pointed out in https://github.com/NixOS/nixpkgs/pull/147307#issuecomment-981094807.

---

Ok, just pushed https://github.com/NixOS/nixpkgs/commit/9ec60f810df8c4b6e0851e00f4a4d86be3340e69 which should resolve the test failures.